README.md

```diff
@@ -19,8 +20,7 @@ or JAX-RS resources.
 
 ### Prerequisites 
 
-Unfortunately, right now, there are no release of this project, so you need to build it by yourself. The build process
- requires Apache Maven and Java 8+.  You need to build the SmallRye Reactive Stream Operators implementation first:
+The build process requires Apache Maven and Java 8+. You need to build the SmallRye Reactive Stream Operators implementation first:
```

Comment:
Actually... this is not true anymore :-)

Reply:
The only requirements are Apache Maven and Java 8+.

Reply:
Done :)